diff -ru nupack/cmake/Options.cmake nupack/cmake/Options.cmake
--- nupack/cmake/Options.cmake	2020-04-23 00:54:49.317527718 -0700
+++ nupack/cmake/Options.cmake	2020-04-23 01:02:24.846447813 -0700
@@ -19,7 +19,7 @@
 option(NUPACK_ONLY               "Compile only a given file"                     OFF)
 option(NUPACK_DESIGN_ONLY        "Compile only given files with design library"  OFF)
 option(NUPACK_PGO                "Enable PGO, can be OFF, READ, or WRITE"        OFF)
-option(NUPACK_EXTERNAL_ARMADILLO "Use external version of armadillo"             OFF)
+option(NUPACK_EXTERNAL_ARMADILLO "Use external version of armadillo"             ON)
 
 ################################################################################
 
diff -ru nupack/external/rebind/setup.py nupack/external/rebind/setup.py
--- nupack/external/rebind/setup.py	2020-04-23 00:54:48.937520277 -0700
+++ nupack/external/rebind/setup.py	2020-04-23 00:55:59.728906490 -0700
@@ -1,9 +1,6 @@
 import os
 from setuptools import setup
 
-def read(fname):
-    return open(os.path.join(os.path.dirname(__file__), fname)).read()
-
 setup(
     name="rebind",
     version="0.0.0",
@@ -13,5 +10,4 @@
     keywords="C++ unit test framework",
     url="http://packages.python.org/an_example_pypi_project",
     packages=['rebind'],
-    long_description=read('README.md'),
 )

